### DISCUSSION TITLE
Data Set for face-api

### DISCUSSION THREAD

**Discussion by naveensiitd** [vladmandic/face-api#137] [Q&A] [answered]
Hi Vladimir,
I refer your github code for face-api.js, so wants to know more about what data used for getting facial emotions.
It would be great if you share some info about data or link to access the data specially images.

Regards,
Naveen Shrivastava

**Comment by vladmandic** [comment, marked as answer]
this version of `face-api` is an updated/modernized version of original `https://github.com/justadudewhohacks/face-api.js` which has not been updated for a long time. original library was trained using:
- <https://ibug.doc.ic.ac.uk/resources/facial-point-annotations/> for landmarks
- <http://vis-www.cs.umass.edu/fddb/> verification and benchmarks
- <https://www.kaggle.com/c/challenges-in-representation-learning-facial-expression-recognition-challenge> expressions

**Reply by naveensiitd** [reply]
Hi Vladimir,
Thanks for your quick response!!
So basically FER2013 dataset used for emotional recognition and I observed that on Model training we can get up to 90% accuracy while on Model testing we can get up to 70% accuracy, this is not overfitting instead of highly imbalance data set on one emotion specifically Disgust emotion is very less as compared to other emotions.

Again thanks!


Regards,
Naveen Shrivastava

**Reply by vladmandic** [reply]
yes, that sounds about right...
